lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
```

Comment:
Since EFS can be expensive we need to make sure that it's tagged with the user's ID

Reply:
EFS is no longer a shared resource.  We now create an EFS for each lambda that is provisioned.  Each EFS will be contain Department and Projects tags.